contrib/job-scripts/coreudmp-handling/README

```diff
@@ -0,0 +1,29 @@
+These are a set of scripts that will:
+
+* Setup the core pattern to the "coredumps" sub directory of the
+  current job and sets the core file size limit to "unlimited"
+* Restore the original settings once the job finishes running tests
+* Links the core files (if any) on the job level "coredumps" subdir
+  into the test result specific "coredumps" dir based on the time
+  stamps of the test directories and the coredump files.
+
+If you system lacks the prlimit utility, then you have to set core
+file size limits yourself.  Options to do so include runnning:
+
+ $ ulimit -c unlimited
+
+Or setting it permanently with an entry in /etc/security/limits.conf,
+or adding a file to /etc/security/limits.d.
+
+To enable these scripts during a development session, set the
```

Comment:
I'd add that to install this permanently, copy the content into `/etc/avocado/scripts/...`. Then you could add this paragraph about temporary usage.